NitroxServer/ConsoleCommands/Abstract/Command.cs

```diff
@@ -131,7 +131,7 @@ public static void SendMessageToPlayer(Optional<Player> player, string message)
         public static void SendMessage(Optional<Player> player, string message)
         {
             SendMessageToPlayer(player, message);
-            Log.Info(message);
+            Log.Info($"{(player.HasValue ? $"=> {player.Value.Name}: ": "")}{message}");
```

Comment:
?

Reply:
this logging was weird before so this lets you understand which message is sent to which player or to the server